api/File.json

```diff
@@ -334,7 +334,7 @@
               "prefix": "webkit"
             },
             "edge": {
-              "version_added": false
+              "version_added": "13"
```

Comment:
`File.prototype.webkitRelativePath` confirmed to exist in Edge 13 but not Edge 12. I don't know how to get a File instance to be really sure, but I think this can be trusted.